packages/react-native-bpk-component-calendar/README.md

```diff
@@ -259,3 +263,30 @@ An any matcher will match if the date is equal to any of the dates provided.
 ```
 
 Returns **DateMatcher** an any date matcher.
+
+### colorBucket
+
+Creates a new color bucket to be used in BpkCalendar.
+
+A color bucket is used to define custom colours for calendar days.
+
+#### Parameters
+
+-   `color` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** The background color
+-   `days` **DateMatcher** The days in this bucket
+-   `textStyle` **TextStyle?** The text style. Valid values are `light` or `dark`.
```

Comment:
Should this be an enum? e.g. `TextStyles.LIGHT`

Reply:
it can but I think is irrelevant if if we use flow and say the type is `light | dark`. I think we had this discussion before, but the object version `TextStyles.LIGHT` has no benefit over strings and is more verbose, specially considering we use flow